src/include/sof/dma.h

```diff
@@ -178,6 +178,11 @@ struct dma {
 	int sref;		/**< simple ref counter, guarded by lock */
 	const struct dma_ops *ops;
 	atomic_t num_channels_busy; /* number of busy channels */
+#ifndef CONFIG_APOLLOLAKE
+	atomic_t mask_irq_channels; /* bitmask of channels with registered IRQs
```

Comment:
atomic types cant be used as a bitmask yet as we have no atomic logical operators yet.